pkg/server/singleprocess/service_project.go

```diff
@@ -245,6 +245,10 @@ func queueInitOps(s *Service, ctx context.Context, project *pb.Project) error {
 }
 
 func projectNeedsRemoteInit(project *pb.Project) bool {
+	if project.State == pb.Project_DESTROYING {
```

Comment:
Not sure if we'll ever make more Project states...but perhaps this could work if it makes sense to protect when we do?
```suggestion
	if project.State != pb.Project_ACTIVE {
```

Reply:
Ah yeah, that makes a lot of sense. I think at the time I wasn’t 100% sure if projects created prior to project destroy being introduced would be marked as active.

Reply:
I believe since it's an Enum in protobuf, it'd take the default state which is `Active`: https://github.com/hashicorp/waypoint/blob/af6141fa752085ec2536bfb3c56a15d60c5c0af5/pkg/server/proto/server.proto#L831-L838 So hopefully yes!

Reply:
Perfect, I’ve committed that suggestion.